lib/capybara/spec/views/with_animation.erb

```diff
@@ -77,6 +63,20 @@
     <a href='#' class='animation' onclick='this.classList.add("away")' oncontextmenu='this.classList.add("pseudo")'>
       animate me away
     </a>
+    <script src="/jquery.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
If a new view is needed for new tests please create it rather than modifying the existing one.

Reply:
🆗